go-controller/pkg/ovn/namespace.go

```diff
@@ -410,14 +412,17 @@ func (oc *DefaultNetworkController) deleteNamespace(ns *kapi.Namespace) error {
 
 // getNamespaceLocked locks namespacesMutex, looks up ns, and (if found), returns it with
 // its mutex locked. If ns is not known, nil will be returned
-func (oc *DefaultNetworkController) getNamespaceLocked(ns string, readOnly bool) (*namespaceInfo, func()) {
+func (nsm *namespaceManager) getNamespaceLocked(ns string, readOnly bool) (*namespaceInfo, func()) {
```

Comment:
would it make sense to move the namespace manager along w/ its receiver funcs to a new file (possibly even a new package ?... ) ?